sonar-jmeter-plugin/pom.xml

```diff
@@ -43,7 +43,7 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
       <version>${slf4j.version}</version>
-      <scope>test</scope>
+      <scope>provided</scope>
```

Comment:
Yes, that's true: we would only need slf4j-api, but since it's scope is only "provided" (meaning it's not added into the plugin jar) I don't see a benefit in adding an separate slf4j-api dependency. It's explicit that way however, so I'll do it (I hope this is what you wanted...).

By the way I was wondering, how it was compiled before without having the slf4j-dependency...